test/unit/api/response/iiif/manifest.test.js

```diff
@@ -121,6 +121,13 @@ describe("Image Work as IIIF Manifest response transformer", () => {
     expect(placeholderCanvas.type).to.eq("Canvas");
   });
 
+  // it("adds excludes placeholderCanvas property on Image canvases if filset does not have width OR height", async () => {
+  //   const { manifest } = await setup();
+  //   const { placeholderCanvas } = manifest.items[0];
+  //   expect(placeholderCanvas.id).to.eq(`${manifest.items[0].id}/placeholder`);
+  //   expect(placeholderCanvas.type).to.eq("Canvas");
+  // });
+
```

Comment:
Do we want to leave this for later?

Reply:
Nope, don't need it. Good catch. That was where I put things initially but then decided a new `it()` altogether,